docs/installation.rst

```diff
@@ -0,0 +1,149 @@
+to ensure that it still works. If you find something wrong, please raise an issue_.
+
+Installing with conda
+'''''''''''''''''''''
+
+Using conda-forge channel
+"""""""""""""""""""""""""
+
+unidist releases can be installed using ``conda`` from the conda-forge channel. Starting from the first 0.1.0 release
+it is possible to install unidist with chosen execution backend(s) alongside. Current options are:
+
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| **Package name in conda-forge** | **Backend(s)**                                      | **Supported OSs**           |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist                         | `Multiprocessing`_, Python                          | Linux, Windows, MacOS       |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-all                     | `Dask`_, `MPI`_, `Ray`_, `Multiprocessing`_, Python | Linux, Windows              |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-dask                    | `Dask`_                                             | Linux, Windows, MacOS       |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-mpi                     | `MPI`_                                              | Linux, Windows, MacOS       |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-ray                     | `Ray`_                                              | Linux, Windows              |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+
+Installing unidist packages from the conda-forge channel can be achieved by adding conda-forge to your channels with:
+
+.. code-block:: bash
+
+  conda config --add channels conda-forge
+  conda config --set channel_priority strict
+
+For installing unidist with dependencies for Dask and MPI execution backends into a conda environment
+the following command should be used:
+
+.. code-block:: bash
+
+  conda install unidist-dask unidist-mpi -c conda-forge
+
+All set of backends could be available in a conda environment by specifying:
+
+.. code-block:: bash
+
+  conda install unidist-all -c conda-forge
+
+or explicitly:
+
+.. code-block:: bash
+
+  conda install unidist-dask unidist-mpi unidist-ray -c conda-forge
+
+Installing from the GitHub master branch
+''''''''''''''''''''''''''''''''''''''''
+
+If you'd like to try unidist using the most recent updates from the master branch, you can
+also use ``pip``.
+
+.. code-block:: bash
+
+  pip install git+https://github.com/modin-project/unidist
```

Comment:
I suggest to add additional example of installing from sources with some set of dependencies ([ray]/[dask] etc).

Reply:
Good catch!